test/e2e/scalenodes.go

```diff
@@ -28,22 +28,13 @@ var _ = Describe("Scale nodes", func() {
 	// nodes to settle after scale down
 	Specify("node count should match the cluster resource and nodes should be ready", func() {
 		ctx := context.Background()
-
-		oc, err := clients.OpenshiftClustersv20200430.Get(ctx, vnetResourceGroup, clusterName)
+		machinesets, err := clients.MachineAPI.MachineV1beta1().MachineSets(machineSetsNamespace).List(ctx, metav1.ListOptions{})
 		Expect(err).NotTo(HaveOccurred())
-
+		
```

Comment:
Need to remove tabs. Linter will probably complain on this one.

I recommend checking editor settings: most editors (vscode is one) allow removing whitespaces at the end of the line automatically. Very useful.

```suggestion

```